src/Promitor.Core.Scraping/Configuration/Model/ResourceType.cs

```diff
@@ -7,6 +7,7 @@ public enum ResourceType
         Generic = 2,
         StorageQueue = 3,
         ContainerInstance = 4,
-        VirtualMachine = 5
+        VirtualMachine = 5,
+        ContainerRegistry = 6
```

Comment:
I'm beginning to wonder if having the explicit values here will hamper merges in the future... thoughts?

Reply:
Good point, maybe better to remove it?

Reply:
Actually I would suggest keeping them in case we want to re-order it and be safe